app/services/github_service.rb

```diff
@@ -1,12 +1,12 @@
 class GithubService
-  def repos
-    repos = conn.get("/user/repos")
+  def repos(token)
+    repos = conn(token).get('/user/repos')
     JSON.parse(repos.body, symbolize_names: true)
   end
 
-  def conn
-    Faraday.new("https://api.github.com") do |req|
-      req.headers["Authorization"] = "token #{ENV["GITHUB_TOKEN"]}"
+  def conn(token)
```

Comment:
This allows us to pass the current users token as an argument so the repos shown reflect the current user.

Reply:
So this is basically what we skipped testing yesterday by specifying which repo gets displayed